ui/hooks/useTokensToSearch.js

```diff
@@ -71,9 +71,11 @@ export function getRenderableTokenData(
     // And since the token.address from allTokens is checksumaddress
     // token Address have to be changed to lowercase when we are using dynamic list
     tokenAddress = useTokenDetection ? address?.toLowerCase() : address;
-    tokenIconUrl = useTokenDetection
-      ? tokenList[tokenAddress]?.iconUrl
-      : `images/contract/${tokenList[tokenAddress].iconUrl}`;
+
+    tokenIconUrl = tokenList[tokenAddress]?.iconUrl;
```

Comment:
It seems that we can also set this on line 65 as
```let tokenIconUrl = tokenList[tokenAddress]?.iconUrl;```

Then can then remove this line and line 68